testing/dart/canvas_test.dart

```diff
@@ -530,7 +540,7 @@ void main() async {
     // Skia renders a tofu if the font does not have a glyph for a character.
     // However, Flutter opts-in to a Skia feature to render tabs as a single space.
     // See: https://github.com/flutter/flutter/issues/79153
-    final File file = File(path.join('flutter', 'testing', 'resources', 'RobotoSlab-VariableFont_wght.ttf'));
+    final File file = File(path.join(_flutterBuildPath, 'flutter', 'third_party', 'txt', 'assets', 'Roboto-Regular.ttf'));
```

Comment:
I would expect this to be in a folder called fixtures.

Reply:
I think you want "./gen/flutter/impeller/fixtures/assets/Roboto-Regular.ttf"

Reply:
But my `deps` is:
```gn
deps = [ "//flutter/third_party/txt:txt_fixtures" ]
```
What does that have to do with the impeller folder?

Reply:
```bash
$ flutter % ls ../out/host_debug_unopt_arm64/gen/flutter/third_party/txt/assets                   
Roboto-Regular.ttf
```
Hmm....

Reply:
I'm not sure if this is the right target specifically, but any test fixtures should be built to a target in the out directory that you can look up

Reply:
Also if the file didn't exist, we could get a FileNotFoundError AFAICT

Reply:
ahh yeah. Hmmm

Reply:
I mean, the decorations _are_ showing up right 😆

Reply:
Yeah, though that's also true if the font doesn't load (hey, I don't make the rules)

I'll take another look in a bit and add some FML_LOG statements and see if there is something obvious going wrong. Probably (?) using a font in a test that doesn't exist should be an error or at the very least an angry log statement.